lib/records/I_RecCore.h

```diff
@@ -138,8 +138,14 @@ RecErrT RecRegisterRawStatUpdateFunc(const char *name, RecRawStatBlock *rsb, int
 // already been taken out for the callback.
 
 // RecSetRecordConvert -> WebMgmtUtils.cc::varSetFromStr()
-RecErrT RecSetRecordConvert(const char *name, const RecString rec_string, RecSourceT source, bool lock = true,
+RecErrT RecSetRecordConvert(std::string_view name, const RecString rec_string, RecSourceT source, bool lock = true,
```

Comment:
Hmmmm.  I still have welts from your determined explanation of why stiring_views should be passed by const reference not by value.  Or is that only if you've written you letter home after moving your coat onto the lower peg because you have a younger brother?

Reply:
I suppose I should be more consistent. I'll take another look.

Reply:
Yes, this should be by CR, because it's only passed on in the function, not used.